src/sentry/models/dsymfile.py

```diff
@@ -171,21 +183,26 @@ class Meta:
         db_table = 'sentry_projectdsymfile'
         app_label = 'sentry'
 
+    __repr__ = sane_repr('object_name', 'cpu_name', 'uuid')
+
     @property
     def dsym_type(self):
         ct = self.file.headers.get('Content-Type').lower()
         return KNOWN_DSYM_TYPES.get(ct, 'unknown')
 
 
-def _create_macho_dsym_from_uuid(project, cpu_name, uuid, fileobj,
-                                 object_name):
+def _create_dsym_from_uuid(project, dsym_type, cpu_name, uuid, fileobj,
+                           basename):
     """This creates a mach dsym file from the given uuid and open file
```

Comment:
minor: Should probably change this comment since it's specifically about `mach`.